cmd/dmverity-vhd/main.go

```diff
@@ -71,22 +74,42 @@ func fetchImageLayers(ctx *cli.Context) (layers []v1.Layer, err error) {
 		return nil, errors.Wrapf(err, "failed to parse image reference: %s", image)
 	}
 
-	var remoteOpts []remote.Option
-	if ctx.IsSet(usernameFlag) && ctx.IsSet(passwordFlag) {
-		auth := authn.Basic{
-			Username: ctx.String(usernameFlag),
-			Password: ctx.String(passwordFlag),
-		}
-		authConf, err := auth.Authorization()
-		if err != nil {
-			return nil, errors.Wrapf(err, "failed to set remote")
+	destination := ctx.String(destinationFlag)
```

Comment:
maybe it's worth adding a global `bool` flag `-l/--local` instead?

Reply:
By global, does that mean you'd like the flag available for the createVHDCommand function as well?

Reply:
yup! I think it'd make sense.

Reply:
so it'd work similar to the `verbose` flag.